api/src/main/java/au/org/aodn/nrmn/restapi/service/validation/DataValidation.java

```diff
@@ -284,4 +291,14 @@ public Collection<SurveyValidationError> checkFormatting(
         return errors.getAll();
     }
 
+
```

Comment:
prefer function change to hasDigitsOfLength(double number, int digits) and move it to a utility package